sdks/python/apache_beam/typehints/native_type_compatibility.py

```diff
@@ -0,0 +1,176 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Module to convert Python's native typing types to Beam types."""
+
+try:
+  import typing
+except ImportError:
```

Comment:
<!--new_thread; commit:86f15f403df6cb7ad3e9d563c56d82cc4c4ac96a; resolved:0-->
Please remove conditional import and add the dependency to setup.py, as suggested, pinned to the minor version >=3.6.0, <3.7.0.